### PR TITLE
missing mod message

### PR DESCRIPTION
**Pull request by Dorbedo**
**When merged this pull request will:**
- removed BIS_fnc_spawn
- sort PREP

### CODE REVIEW

**Comment by dedmen** [comment]
Can you include missing semicolon in https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/fnc_radioToRequestCount.sqf#L55 ?

File: addons/core/functions/server/fnc_missingModMessage.sqf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  Public: No
+*/
+
+if(isServer) exitWith {};
+waitUntil {sleep 0.1;time > 3};
+if !(isClass(configFile >> "CfgPatches" >> "tfar_core")) exitWith {
+    [player, format["%1: LOOKS LIKE TASK FORCE RADIO ADDON IS NOT ENABLED OR VERSION LESS THAN 1.0", name player]] remoteExec ["globalChat", -2];
+    if (isNull (uiNamespace getVariable ["BIS_fnc_arsenal_cam", objNull])) then {
+        ["LOOKS LIKE TASK FORCE RADIO ADDON IS NOT ENABLED OR VERSION LESS THAN 1.0"] call "BIS_fnc_guiMessage";
```
Comment (jokoho48) [inline addons/core/functions/server/fnc_missingModMessage.sqf]: since when does call support a string right handed?

Comment (dedmen) [reply addons/core/functions/server/fnc_missingModMessage.sqf]: it doesn't

**Comment by dedmen** [comment]
Also replaced the remoteExec with only local. Or do you thing everyone should get a message when someone is missing TFAR?

**Comment by jokoho48** [comment]
this was always very helpful as server admin when you get the information that someone joined with the wrong TFAR version

**Comment by dedmen** [comment]
brawl!

**Pull request merged**